modules/stanford_basic_page_types/templates/components/basic_page_types_list/basic-page-types-list.ui_patterns.yml

```
@@ -0,0 +1,35 @@
+basic-page-types-list:
+  label: Basic Page Types List
+  description: "Building block component for the basic page types list page."
+  fields:
+    basic_page_types_category:
```

we're already in the template, there's no need to prefix every field in the template
```suggestion
    category:
```

---

fixed throughout the template.